samples/rtsp_cam_compatibility_test/docker-compose.l4t.yml

```diff
@@ -0,0 +1,41 @@
+version: "3.3"
+services:
+
+  rtsp-city-traffic:
```

Comment:
Let's remove the `city-traffic` part from services' names.

Reply:
done, good catch!